platform-sdk/docs/proposals/TSS-Ledger-Id/tss-ledger-id.md

```diff
@@ -0,0 +1,73 @@
+## Interactions
```

Comment:
I miss 2 sections that should be part of a proposal:
- **Next steps**: This section describes what can be done once the proposal is implemented. Example: _Once this change is done we can define a proposal to create new additions to the plugin-structure for external parties._
- **Alternatives**: Here we give a short overview if the goal of the proposal could be done in a different way and why we not do that. Example: _The same functionallity could be created by using bytecode manipulation at runtime. Since we do not want to depend on manipulation of bytecode in general (see doc XYZ) and using it at runtime would end in bad performance we decided to use code generation at buildtime as described in this proposal._

Reply:
Can you raise these suggestions in slack in the #platform-definitions channel so they can be discussed?

Reply:
The alternatives can be described in the Design Decisions section.